addons/payment_stripe_sca/controllers/main.py

```diff
@@ -0,0 +1,60 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+import logging
+import werkzeug
+
+from odoo.http import route, request
+from odoo.addons.payment_stripe.controllers.main import StripeController
+
+_logger = logging.getLogger(__name__)
+
+
+class StripeControllerSCA(StripeController):
+    _success_url = '/payment/stripe/success'
+    _cancel_url = '/payment/stripe/cancel'
+
+    @route(['/payment/stripe/success', '/payment/stripe/cancel'], type='http', auth='public')
```

Comment:
same route? at least I would send the cancelled payment to the payment page again instead of '/'